primitives/src/policy.rs

```diff
@@ -121,13 +121,13 @@ impl Policy {
 
     /// This is the number of Lunas (1 NIM = 100,000 Lunas) created by millisecond at the genesis of the
     /// Nimiq 2.0 chain. The velocity then decreases following the formula:
-    /// Supply_velocity (t) = Initial_supply_velocity * e^(- Supply_decay * t)
-    /// Where e is the exponential function and t is the time in milliseconds since the genesis block.
-    pub const INITIAL_SUPPLY_VELOCITY: f64 = 875.0;
+    /// Supply_velocity (t) = Initial_supply_velocity * 2^(- Supply_decay * t)
+    /// Where t is the time in milliseconds since the genesis block.
+    pub const INITIAL_SUPPLY_VELOCITY: f64 = 0.10594132556065439;
```

Comment:
Is this a number that is dependent on the PoS genesis block timestamp? Or can we fix it to a stable timestamp, e.g. PoW genesis timestamp?

Reply:
Yes, if we want to 'continue' the curve, we would need to use the PoW timestamp

Reply:
Done